teradatalabs/mapr52-hive/files/bootstrap.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+#configure mapr
+/opt/mapr/server/configure.sh  -N mycluster -Z localhost -C localhost -HS localhost -no-autostart
+
+#setup disk for mapr
+dd if=/dev/zero of=/home/mapr/storagefile bs=1G count=10
+/opt/mapr/server/disksetup -M -F /root/disk.txt
+
+#create proxy setup for hive users
+chmod 755 /opt/mapr/conf/proxy
+touch /opt/mapr/conf/proxy/hive-user
+touch /opt/mapr/conf/proxy/hdfs-user
+
+#start zookeeper
+service mapr-zookeeper start
+
+#start warden 
+service mapr-warden start
+
+
+#configure hive
+/opt/mapr/server/configure.sh -R
+
+#wait for warden to start all the services
+
+sh /root/wardenTracker.sh
+
+
+#run hdfs commands
+hadoop fs -mkdir /user/hive-user
```

Comment:
Can you combine these commands 
hadoop fs -mkdir /user/hive-user /user/hdfs-user